docs/_replacements/docs/apis/fluid-framework/index.md

```diff
@@ -8,10 +8,8 @@ area: "libraries"
 summary: "The fluid-framework package bundles a collection of Fluid Framework client packages for easy use when paired with a corresponding service client package (for example, @<!-- -->fluidframework/azure-client or @<!-- -->fluidframework/tinylicious-client)."
 children:
   - '@fluidframework/map'
-  - '@fluidframework/sequence'
```

Comment:
Removes the sequence API docs from appearing in navigation from the fluid-framework package API docs landing page on the public website.